src/Components/buttons/index.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 
-const CreateBookButton = () => {
+const CreateBookButton = ({click}) => {
```

Comment:
there should be spacing before and after the curly braces. the prop name should be `onClick` too.